RestingStateStats/RestingStateStats.sh

```diff
@@ -957,13 +957,13 @@ main()
 		0)
 			# Use Compiled Matlab
 			matlab_exe="${HCPPIPEDIR}"
-			matlab_exe+="/RestingStateStats/Compiled_RestingStateStats/distrib/run_RestingStateStats.sh"
+			matlab_exe+="/RestingStateStats/Compiled_RestingStateStats/run_RestingStateStats.sh"
 
 			# TBD: Use environment variable instead of fixed path
 			if [ "${CLUSTER}" = "1.0" ]; then
-				matlab_compiler_runtime="/export/matlab/R2013a/MCR"
+				matlab_compiler_runtime="/export/matlab/R2016b/MCR"
```

Comment:
SetUpHCPPipeline.sh has MATLAB_COMPILER_RUNTIME in it, which should probably be used as-is here.  You can modify the wustl-specific SetUp... to have the CLUSTER variable logic in it, and thus remove CLUSTER from all the pipelines scripts.

Reply:
Thanks. Updated to use MATLAB_COMPILER_RUNTIME.

The CLUSTER variable is not likely to be needed anymore. It was introduced when the CHPC was in the change over from the "old" cluster to the "new" cluster and things were installed in different locations depending upon the cluster. The "old" cluster is dead, and changing the value of the MATLAB_COMPILER_RUNTIME variable in the SetUpHCPPipeline.sh file should be sufficiently flexible.

The CLUSTER variable is also likely to be confusing to external users.

Reply:
I think that eliminates the last remaining use of the CLUSTER variable in the pipeline scripts.